conductor-client/src/main/kotlin/com/openlattice/authorization/HazelcastAuthorizationService.kt

```diff
@@ -426,6 +427,7 @@ class HazelcastAuthorizationService(
     @Timed
     override fun getAllSecurableObjectPermissions(keys: Set<AclKey>): Set<Acl> {
         return aces.entrySet(hasAnyAclKeys(keys))
+                .filterNot { it.value.isEmpty() }
```

Comment:
the most minor of nits, but I think
```
.filter { it.value.isNotEmpty() }
```
feels more natural and is a bit easier to read than
```
.filterNot { it.value.isEmpty() }
```

Reply:
I agree, that's much better; didn't know `isNotEmpty()` was available.